src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -10,16 +10,70 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
 -->
 <Project>
 
-  <!-- workaround https://github.com/NuGet/Home/issues/4726 -->
-  <PropertyGroup>
-    <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
-  </PropertyGroup>
-
   <PropertyGroup Condition=" '$(IncludeSymbols)' != 'false' ">
     <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);_EnsureDebugTypeIsPortable</GenerateNuspecDependsOn>
+    <PackDependsOn Condition="'$(NuspecFile)' != ''">$(PackDependsOn);_GenerateSymbolsNupkg</PackDependsOn>
+
+    <!-- workaround https://github.com/NuGet/Home/issues/4726 -->
+    <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
+    <IncludeSource Condition="'$(NuspecFile)'!=''">false</IncludeSource>
+
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);_EnsureDebugTypeIsPortable</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
 
+  <Target Name="_GenerateSymbolsNupkg" >
+    <Error Condition="'$(NuspecFile)' != '' AND '$(SymbolsNuspecFile)' == ''"
```

Comment:
Thoughts? We want to ensure we have symbols when we have a nuspec and we want to avoid putting pdbs in the main nupkg. This moves the symbol generation \ verifcation in to the Sdk.

Reply:
Implementation looks fine, but I'm concerned about the longevity of this solution. The `PackTask` is "pubternal" API. Even though it's accessible, I've been told not to use it directly as the NuGet team regularly make changes to this task and targets.

Reply:
Aww. We could write a task to call `PackageBuilder` ourselves, Cleans up some of the copying stuff

Reply:
We have the PackNuspec task in KoreBuild that uses PackageBuilder. This may become part of Arcade https://github.com/dotnet/arcade/pull/57, or part of NuGet itself if I had my wishes. But, it's probably easier to go with this for now. I'm okay with this change as long as you're aware it's a thing we'll likely have to service.

Reply:
Let me have a look at `PackNuspec`. `PackTask` has some quirky behavior that would be nice to avoid.